server/src/helpers/utils.helper.ts

```diff
@@ -236,6 +236,13 @@ export function isTooljetVersionWithNormalizedAppDefinitionSchem(version) {
   return semver.satisfies(semver.coerce(version), '>= 2.24.0');
 }
 
+/**
```

Comment:
Shall we create a new util file for component related functions?
`components.helper.ts`

Reply:
This functions is no longer required so removed it.